security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -33,6 +38,17 @@ The following advisory is available for the OpenShift Compliance Operator 0.1.52
 
 * Previously, the Machine Config Operator used `base64` instead of `url-encoded` code in the latest release, causing Compliance Operator remediation to fail. Now, the Compliance Operator checks encoding to handle both `base64` and `url-encoded` Machine Config code and the remediation applies correctly. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2082431[*BZ#2082431*])
 
+[id="compliance-operator-0-1-52-known-issue"]
+== Known issue
+
+* When `"debug":true` is set within the scansettingbinding, the pods generated by scansettingbinding are not removed when the scansettingbinding is deleted. As a workaround, the following command will delete the remaining pods:
```

Comment:
In our docs, the word "scansettingbinding" is currently written several ways: 

`ScanSettingBinding` (with backticks + capitalized)
scansettingbinding (like you have)
ScanSettingBinding (no backticks) 

This might be context dependent, but I'm not sure personally. In your example, is it like...

When `"debug":true` is set within the `ScanSettingBinding` object. . ." ? 

tl;dr--just bringing this up for consideration.